S4TF_Tutorial_2.ipynb

```diff
@@ -210,7 +210,7 @@
         "\n",
         "### Select the model\n",
```

Comment:
Above this, I'd replace "For the Celsius to Fahrenheit Conersion problem, the model defines the relationship between the Celsius Temperature and Fahrenheit Temperatures."

with

"For the Celsius to Fahrenheit conversion problem, the model defines the relationship between the Celsius temperatures and Fahrenheit temperatures."